sycl/test/on-device/basic_tests/stream/stream_flush_buf_init_dep.cpp

```diff
@@ -0,0 +1,36 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: env SYCL_DEVICE_TYPE=HOST SYCL_PRINT_EXECUTION_GRAPH=after_addCG %t.out
+// RUN: cat graph_2after_addCG.dot | FileCheck %s
+// RUN: rm *.dot
```

Comment:
I believe it's much better to add a unit-test instead of validating proper order via graph output.

Reply:
Done